.travis.yml

```diff
@@ -1,32 +1,39 @@
 language: node_js
 node_js:
-  - "lts/*"
+  - lts/*
 install:
-  - npm install 
+  - npm install
   - lerna bootstrap
 dist: trusty
 sudo: false
 env:
   global:
     - CC_TEST_REPORTER_ID=08a773cb4ea5811add5a45e12873e5cd2634c005568705cc37abfd5217617a32
 before_script:
-  # Commented out for local testing only on MacOS
-  #- curl -L https://codeclimate.com/downloads/test-reporter/test-reporter-latest-darwin-amd64 > ./cc-test-reporter
   - curl -L https://codeclimate.com/downloads/test-reporter/test-reporter-latest-linux-amd64 > ./cc-test-reporter
   - chmod +x ./cc-test-reporter
-  - ./cc-test-reporter before-build
+  - "./cc-test-reporter before-build"
 script:
   - lerna run test --stream
-
 after_script:
   # See https://github.com/codeclimate/test-reporter/issues/263
-  - if [ "$TRAVIS_PULL_REQUEST" == "false" ]; then
+  - if [ "$TRAVIS_PULL_REQUEST" == "false" ]; then 
       for f in packages/*; do
-        if [ -d $f ]; then
-          echo $f;
-          ./cc-test-reporter format-coverage -t lcov -o coverage/coverage.${f//\//-}.json $f/coverage/lcov.info;
-        fi
+        if [ -d $f]; then
+          echo $f; 
+          ./cc-test-reporter format-coverage -t lcov -o coverage/coverage.${f//\//-}.json $f/coverage/lcov.info; 
+        fi 
       done;
-      ./cc-test-reporter sum-coverage -o coverage/coverage.total.json coverage/coverage.*.json;
-      ./cc-test-reporter upload-coverage -i coverage/coverage.total.json;
+        ./cc-test-reporter sum-coverage -o coverage/coverage.total.json coverage/coverage.*.json; 
+        ./cc-test-reporter upload-coverage -i coverage/coverage.total.json;
     fi
+deploy:
+  provider: npm
+  email: "EMAIL_ADDRESS"
+  api_key:
+    secure: AQpAGAHolfKGOkO/RCPWF0nt4W5h43NNZJ0sIjPD6rHP7uqd9I/VAU6gYHTgWW0aLvP+OD/wFYMTP9M3U5VSdPBH1O+nTZW/iB/jTdxfJnbJ+yx/KtrgShur+V+s+ldLUQoMX//F7G/ljnknV34JAv01L4veeFkytthuca/9eMEYAnRjMnMyUSJApBli6V2lVK1O/xvyOcmPnhz1llWKgZYXjs7mwVwNP5xzFsmuFYCt6Ryn5tf3yj88M6mLyEzqKtb9HRPFxdflFj4zweG1AqvNRkU1lllfV/FWs3j1bfZRPVA1DmaYFKmtplKAKdHFUZNU8I2RTB+5z8btAxmQYw2BNhaxRWuxgFHA3NjG3pOecWVy3rhEUObuaavXqXHrnc/28H43nGM3nH3EbvmsLXGHl6gBlXe4q1xoYVHjDGU7fCngvOXU1zDIoXwDnb9hl76gnUPrdX8WTjhAUYwgD9YzdS0C5IP+8K2QeiVzvyT5pZUEOzkKiB39y+Fp9GYRjOVFcaLh0c/PuYnR0NIfQM6RRyvbupUG9vCz6cXXLPnHPtiAImYeSXkNQ3zV2OlUP2MF4J+cQfQkN5o4lGmgrEeJbgSfqudRUEpRSZflEuLdywC9XSngHbv9onxXryGBkiM3U3EXuqkClEutQrdDbb68Ty/mc/QyHeMo2IpJ87c=
+  script: 
+    - lerna run publish --conventional-commits --yes
```

Comment:
is this going to publish *every* package?

Reply:
This will publish all packages that require changes or have changed with the exception of private packages.

details on lerna publish here: https://github.com/lerna/lerna#publish

As an "FYI":
- conventional-commits flag will generate the changelog according to semantic versioning. 
- ```-- yes``` bypasses the check to confirm you want to publish packages with changes since we are running this in a CI environment